requirements/local.txt

```diff
@@ -18,3 +18,10 @@ pytest-sugar==0.8.0
 
 # Own stuff
 django-rosetta==0.7.12
+
+
+# REST-Framework
+
+djangorestframework==3.3.0
+markdown==2.1.0
+django-filter==1.0.1
```

Comment:
Aber macht es nicht sinn, es direkt in `base.txt` zu tun, weil wir es dann auch auf der Staging Version testen können?